### PR TITLE
Initial Pull Request - Yi 

### PR DESCRIPTION
**Pull request by yizucodes**
Hi @flyingant,

Please checkout the README file before reviewing the app. 
Thank you for your time and looking forward to your feedback!

Best,
Yi

### CODE REVIEW

File: finale/src/components/district.jsx [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import Township from './township';
+import data from "../data"
+import dl_logo from "../img/download_icon.png"
+
+class District extends Component {
```
Comment (flyingant) [inline finale/src/components/district.jsx:6]: actually, right now you could try to transfer the react component like this to the real Pure function with hooks which is more elegant.

File: finale/src/styles/App 2.scss [hunk 2]
```diff
@@ -0,0 +1,37 @@
+$backgroundColor: #585a6f;
```
Comment (flyingant) [inline finale/src/styles/App 2.scss]: Please remove all files appended with `filename 2.xx`, it's unaccepted

**Comment by flyingant** [comment]
@yizu100 I would send some feedback to our recruitment team.
Of course you can continue to hack this if you want.

File: finale/src/styles/App 2.scss (hunk 2 above)
Comment (yizucodes) [reply finale/src/styles/App 2.scss]: Removed. Thank you for mentioning this.

File: finale/src/components/district.jsx (hunk 1 above)
Comment (yizucodes) [reply finale/src/components/district.jsx:6]: Do you mean doing something similar to **Adding Hooks to Classes with State** from [this article](https://www.digitalocean.com/community/tutorials/five-ways-to-convert-react-class-components-to-functional-components-with-react-hooks) ?

Comment (flyingant) [reply finale/src/components/district.jsx:6]: Yep, just tried the `useState `

Comment (yizucodes) [reply finale/src/components/district.jsx:6]: Ok will try it to create the logic for expanding the collapsible rows.

Comment (yizucodes) [reply finale/src/components/district.jsx:6]: Hi @flyingant,

Hope you had a great weekend! I created a separate component called ```DistrictHook``` for the purpose of this challenge and used the useState hook to expand and collapse township rows. This component will render when you select ```District``` in the dropdown filter. 

Looking forward to your feedback. 

Best,
Yi

Comment (flyingant) [reply finale/src/components/district.jsx:6]: Where is it?

Comment (yizucodes) [reply finale/src/components/district.jsx:6]: Hi @flyingant,
You can find the ```DistrictHook``` component [here](https://github.com/yizu100/test-frontend/blob/master/finale/src/components/districtHook.jsx)

**Pull request closed without merging**